src/sql/finalize.sql

```diff
@@ -593,6 +598,42 @@ $$;
 CREATE FUNCTION alter_vector_index("index" OID, "key" TEXT, "value" TEXT) RETURNS void
 STRICT LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_alter_vector_index_wrapper';
 
+CREATE FUNCTION vector_dims("v" vector) RETURNS INT
+STRICT PARALLEL SAFE LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_vector_dims_wrapper';
+
+CREATE FUNCTION vector_norm("v" vector) RETURNS real
+STRICT PARALLEL SAFE LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_vector_norm_wrapper';
+
+CREATE FUNCTION vector_add("v1" vector, "v2" vector) RETURNS vector 
+STRICT PARALLEL SAFE LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_vecf32_operator_add_wrapper';
```

Comment:
Why don't we just use `+`?

Reply:
Such sql don't works for me. Is there any better way?
```
CREATE AGGREGATE sum(vector) (
	SFUNC = +,
	STYPE = vector,
	COMBINEFUNC = +,
	PARALLEL = SAFE
);
```

Reply:
```
CREATE AGGREGATE sum(vector) (
        SFUNC = _vectors_vecf32_operator_add,
        STYPE = vector,
        COMBINEFUNC = _vectors_vecf32_operator_add,
        PARALLEL = SAFE
);
```

It works for me.

Reply:
get! thanks